torch/_inductor/scheduler.py

```diff
@@ -823,57 +838,29 @@ def get_estimated_runtime(self) -> float:
         except Exception:
             return 0
 
-        if isinstance(self, ExternKernelSchedulerNode):
-            assert isinstance(self.node, ir.ExternKernel), f"{type(self.node)=}"
-            op = kernel_name_to_op.get(
-                getattr(self.node, "python_kernel_name", ""), None
+        if isinstance(self, FusedSchedulerNode):
+            flops_est: int | None = sum(
```

Comment:
I mentioned this in the other review - if you have `convolution` and we permute the input to channels last, sometimes the origins will be Convolution. can you make sure we do not include Flops if a node is not a template, and not an extern kernel ?